src/components/date-picker/date-picker.tsx

```diff
@@ -48,6 +47,7 @@ const nativeFormatForType = {
  * @exampleComponent limel-example-date-picker-year
  * @exampleComponent limel-example-date-picker-formatted
  * @exampleComponent limel-example-date-picker-programmatic-change
+ * @exampleComponent limel-example-date-picker-composite
```

Comment:
I think we need to add `style="--example-controls-column-layout:auto-fit;"` to the example-control element in this composite example 😅

<img width="544" alt="image" src="https://user-images.githubusercontent.com/154725/210365025-d145b20e-33e3-4733-800b-feddac7146a4.png">